buildSrc/private/src/main/kotlin/androidx/build/AndroidXImplPlugin.kt

```diff
@@ -510,11 +510,12 @@ class AndroidXImplPlugin @Inject constructor(val componentFactory: SoftwareCompo
             }
         }
 
-        // Standard docs, resource API, and Metalava configuration for AndroidX projects.
-        project.configureProjectForApiTasks(
-            LibraryApiTaskConfig(libraryExtension),
-            androidXExtension
-        )
+// commenting, as the jb-fork doesn't have API checks yet, and this can break build in certain cases
```

Comment:
1. Let's add clear reason why it's commented - version number check.
2. Maybe removing instead of commenting will work better to mitigate conflicts (in case of updating buildSrc)

Reply:
Let's just remove it. Done.